private_repo/ansible/inventory/dev/Core/common.yml

```diff
@@ -20,41 +20,78 @@ sunbird_mail_server_from_email: "support@myorg.com"       # Email ID that should
 # List of mail ids to whome the monitoring alerts should be sent.
 alerts_mailing_list : "devops@myorg.com"                         # Comma separated mail list for Alerts; eg: user1@mail.com, user2@mail.com
 
+# Cloud Service Provider Variables
+cloud_public_storage_accountname: ""  # If cloud_service_provider is aws--> access key, gcp--> service account name, azure--> storage account name
```

Comment:
Please add more detailed examples on how to define a var. Example - key format, values etc